script/test/test_get_pkgs/win32/BraveBrowserBetaSetup.exe

```diff
@@ -0,0 +1 @@
+BraveBrowserBetaSetup_70_0_56_8.exe
```

Comment:
what are these files? They are named as .exe, but seem to be text files that contain the name of an exe?

Reply:
they're just dummy files to test upload